tests/test_baseband_readout.py

```diff
@@ -14,11 +15,19 @@
 
 from kotekan import runner
 
-
 # Skip if HDF5 support not built into kotekan
 if not runner.has_hdf5():
     pytest.skip("HDF5 support not available.", allow_module_level=True)
 
+def is_docker():
+    path = '/proc/self/cgroup'
+    return (
+        os.path.exists('/.dockerenv') or
+        os.path.isfile(path) and any('docker' in line for line in open(path))
+    )
+
+if is_docker():
+    pytest.skip("Does not work in Github Actions docker run.", allow_module_level=True)
```

Comment:
Would it be worth it to see if Jenkins can survive a single build to run the tests disabled here?

Reply:
Nope. Not interested in wasting my time debugging Jenkins either.